### PR TITLE
Document Update Actor Flags

### PR DESCRIPTION
**Pull request by engineer124**
The final 2 actor flags for MM. Both involve flags that affect when an actor can update. 

One is tied to both the Song of Soaring and Song of Time cutscene. I couldn't think of a single word/term that encapsulates both of these cutscenes/events, so I just included both.

The other is tied to the freeze tied to `sCategoryFreezeMasks`, and this flag provides an exception to allow the actor to continue updating.

### CODE REVIEW

**Review by AngheloAlf — APPROVED** [review]

File: src/overlays/actors/ovl_En_Bom/z_en_bom.c [hunk 1]
```diff
@@ -301,11 +301,11 @@ void EnBom_Move(EnBom* this, PlayState* play) {
 }
 
 void EnBom_WaitForRelease(EnBom* this, PlayState* play) {
-    // if parent is NULL bomb has been released
+    // if parent is NULL if bomb has been released
```
Comment (hensldm) [inline src/overlays/actors/ovl_En_Bom/z_en_bom.c:304]: Having a hard time parsing this. Should the first if be removed so it reads
"parent is NULL if bomb has been released"?

**Review by hensldm — COMMENTED** [review]
Nice. Just one small comment

File: src/overlays/actors/ovl_En_Bom/z_en_bom.c (hunk 1 above)
Comment (fig02) [reply src/overlays/actors/ovl_En_Bom/z_en_bom.c:304]: its "if parent pointer is null, the bomb has been released"
I wrote this comment years ago at this point when decompiling the bomb actor lol, so my bad

Comment (fig02) [reply src/overlays/actors/ovl_En_Bom/z_en_bom.c:304]: what you said works too if you remove the leading "if"

**Review by hensldm — APPROVED** [review]

**Pull request merged**